future/src/ct/ct_actions_others.cc

```diff
@@ -158,7 +161,7 @@ void CtActions::embfile_open()
 
     spdlog::debug("embfile_open {}", static_cast<std::string>(filepath));
 
-    CtFileSystem::external_filepath_open(filepath, false);
+    CtFileSystem::external_filepath_open(filepath.c_str(), false, _pCtMainWin->get_ct_config());
```

Comment:
Why .c_str()? Maybe .string()?

Reply:
`filepath` is a `Glib::ustring` which I didnt change as I mentioned previously